codebuild_test.go

```diff
@@ -1,42 +0,0 @@
-/*
- Copyright 2021 The GoPlus Authors (goplus.org)
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
-     http://www.apache.org/licenses/LICENSE-2.0
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
-*/
-
-package gogen
-
-import (
-	"go/token"
-	"go/types"
-	"testing"
-)
-
-func TestCircularEmbeddedFieldLookup(t *testing.T) {
```

Comment:
why to remove this test case?

Reply:
invalid typeA and typeB can check by validType.